tests/publiccloud/az_accelerated_net.pm

```diff
@@ -23,14 +22,16 @@ Creates a VM in Azure and installs IPERF binaries on it
 =cut
 sub prepare_vm {
     my ($self, $provider) = @_;
-    my $iperf = get_required_var('IPERF_FILE');
     record_info('INFO', 'Create VM');
-    my $instance = $provider->create_instance(check_guestregister => 0);
-    record_info('Instance', 'Instance ' . $instance->instance_id . ' created');
-    record_info('Iperf', 'Install IPerf binaries in VM');
-    $instance->run_ssh_command(cmd => "wget https://iperf.fr/download/opensuse/$iperf");
-    $instance->run_ssh_command(cmd => "sudo rpm -i  $iperf");
-    return $instance;
+    my @instances = $provider->create_instances(check_guestregister => 0, count => 2);
+    foreach my $instance (@instances) {
+        record_info('Instance', 'Instance ' . $instance->instance_id . ' created');
+        record_info('Iperf', 'Install IPerf binaries in VM');
+        $instance->run_ssh_command(cmd => "sudo zypper -n ar --no-gpgcheck https://download.opensuse.org/repositories/network:/utilities/15.5/ net_perf");
```

Comment:
I wonder if it would help having this repo in an external variable, as there is also iperf in `https://download.opensuse.org/repositories/network:/utilities/15.4/`. Can this have dependency issues on several SP versions?

Reply:
That makes sense, anyway the sle15sp5 builds are missing there, so maybe we should contact the owner.